e2e/terraform/network.tf

```diff
@@ -45,7 +45,7 @@ resource "aws_security_group" "primary" {
   # Consul
   ingress {
     from_port   = 8500
```

Comment:
Can this be changed also to `8501`?

Reply:
Unlike Nomad, Consul uses separate ports for HTTP vs HTTPs, and convention is to use 8500 for plaintext and 8501 for TLS. As TLS is optional, I exposed both ports. Will add a comment.